lib/transactions.js

```diff
@@ -99,6 +108,25 @@ TxController.prototype.transformTransaction = function(transaction, options, cal
 
   transformed.txlock = transaction.txlock;
 
+  if (transaction.proRegTx !== undefined) {
+    transformed.preRegTx = transaction.preRegTx;
+  }
+  if (transaction.proUpServTx !== undefined) {
```

Comment:
Might be possible to put these in a loop, but not sure how JS works well enough right now, would have to test. E.g. (pseudocode):

```
for thing in ["proRegTx", "proUpServTx", etc] {
  if transaction.defined(thing) {
    transformed.thing = transaction.thing;
  }
}
```

Something like that might help to prevent typos like the above ^.

Reply:
I'd be happy to see something like this, but I'm having some issues understanding all the quirks in JS when it comes to objects and behavior of undefined stuff, so I prefer to keep it simple and revert to repeating lines...If someone with more experience in JS can change this, I'm happy to get rid of the current code.